packages/test-runner-intercept/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@web/test-runner-intercept",
+  "version": "0.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Plugin for intercepting modules in @web/test-runner",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/modernweb-dev/web.git",
+    "directory": "packages/test-runner-intercept"
+  },
+  "author": "modern-web",
+  "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/test-runner-intercept",
+  "main": "browser/index.js",
+  "exports": {
+    ".": {
+      "types": "./browser/index.d.ts",
+      "default": "./browser/index.mjs"
+    },
+    "./plugin": {
+      "types": "./plugin.d.ts",
+      "import": "./plugin.mjs",
+      "require": "./dist/interceptModulePlugin.js"
+    }
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "mocha test/**/*.test.ts --require ts-node/register",
+    "test:watch": "mocha test/**/*.test.ts --require ts-node/register --watch --watch-files src,test"
+  },
+  "files": [
+    "*.d.ts",
+    "*.js",
+    "*.mjs",
+    "dist",
+    "src"
+  ],
+  "keywords": [
+    "web",
+    "dev",
+    "server",
+    "test",
+    "runner",
+    "testrunner",
+    "module",
+    "interception",
+    "intercept"
+  ],
+  "dependencies": {
+    "@web/dev-server-core": "^0.7.0",
+    "es-module-lexer": "^1.3.1"
+  },
+  "devDependencies": {
+    "@web/test-runner": "^0.18.0",
```

Comment:
you shouldnt have to add these dev deps here if they're already part of the monorepo package.json right?

Reply:
Do you want me to use relative dependencies instead? I see both ways throughout the repo.

Reply:
I have updated the devDependencies to use the one that was used (`@web/test-runner-core` instead of `@web/test-runner`). This is similar to the deps of eg. `@web/dev-server-rollup`).

Reply:
No I mean, because we're in a monorepo, and we have mocha and chai installed in the monorepo root's package.json, we shouldnt have to also add them explicitly here. This also shouldnt be needed for `@web/dev-server-rollup`

Reply:
This is now updated. :-) Can you check and resolve if ok?